src/ngraph/op/convolution.cpp

```diff
@@ -231,12 +245,19 @@ void op::v1::ConvolutionBackpropData::validate_and_infer_types()
                           delta_shape,
                           ").");
 
-    set_output_type(0, forward_result_et, m_data_batch_shape);
+    set_output_type(0, forward_result_et, get_data_batch_shape());
```

Comment:
set shape to PartialShape::dynamic() if data_shape is not available. Add shape relevancy hints.